pkg/operation/cloudbotanist/alicloudbotanist/controlplane.go

```diff
@@ -251,7 +251,7 @@ func (b *AlicloudBotanist) GenerateCSIConfig() (map[string]interface{}, error) {
 		"enabled":           true,
 	}
 
-	return b.ImageVector.InjectImages(conf, b.ShootVersion(), b.ShootVersion(),
+	return b.InjectSeedShootImages(conf,
```

Comment:
change it to ```InjectShootShootImages```

Reply:
Very Good catch! That must've slipped through... I'll make sure I recheck all of the changes.